guiEditor/src/diagram/guiNode.ts

```diff
@@ -102,19 +110,23 @@ export class GUINode {
     public constructor(globalState: GlobalState, public guiControl: Control) {
         this._globalState = globalState;
         this._ownerCanvas = this._globalState.workbench;
-        
+        this.x = guiControl.leftInPixels;
+        this.y = guiControl.topInPixels;
         guiControl.onPointerUpObservable.add(evt => {
             this.clicked = false;
             console.log("up");
         });
 
         guiControl.onPointerDownObservable.add( evt => {
+            if(!this._ownerCanvas.isUp) return;

```

Comment:
Our tslint/prettier should prevent this from passing lint, but we are not yet linting those files. Will you be able to run a formatter (prettier?) on these files?